t/01_x11utils.t

```diff
@@ -0,0 +1,43 @@
+use strict;
+use warnings;
+use Test::More;
+use Test::Exception;
+use Test::Warnings;
+use testapi qw(set_var);
+use version_utils qw(is_sle is_leap);
+
+
+subtest 'Expected_terminals' => sub {
+    use x11utils 'default_gui_terminal';
```

Comment:
I'd move this to the top. (Perl is still evaluating this at compile time anyway.)